metricbeat/module/elasticsearch/test_elasticsearch.py

```diff
@@ -27,10 +29,36 @@ def test_metricsets(self, metricset):
         """
         elasticsearch metricset tests
         """
+        if metricset == "ml_job":
+            self.create_ml_job()
+
         es = Elasticsearch(self.get_hosts())
         es.indices.create(index='test-index', ignore=400)
         self.check_metricset("elasticsearch", metricset, self.get_hosts(), self.FIELDS + ["service.name"])
 
     def get_hosts(self):
         return [os.getenv('ES_HOST', 'localhost') + ':' +
                 os.getenv('ES_PORT', '9200')]
+
+    def create_ml_job(self):
+        es = Elasticsearch(self.get_hosts())
+
+        # Enable xpack trial
+        try:
+            es.transport.perform_request('POST', "/_xpack/license/start_trial?acknowledge=true")
+        except:
+            print "Trial already enabled"
+
+        # Check if an ml job already exists
+        response = es.transport.perform_request('GET', "/_xpack/ml/anomaly_detectors/_all/")
+        if response["count"] > 0:
+            return
+
+        file = os.path.join(self.beat_path, "module", "elasticsearch", "ml_job", "_meta", "test", "test_job.json")
```

Comment:
Could we use this file also for the integration test?

Reply:
done